### DISCUSSION TITLE
Is Nerdminer v2 Supporting T-Display S3 AMOLED H681 Touch?

### DISCUSSION THREAD

**Discussion by bitcoinservicesag** [BitMaker-hub/NerdMiner_v2#268] [General]
![Uploading Screenshot 2023-12-14 at 13.44.51.png…]()
Hi all, i tried to flash the Nerdminer Software to my new LILYGO T-Display S3 AMOLED Touch (H681). I used the website https://flasher.bitronics.store/ and i get a confirmation that it was flashed successfully. But if i unplug and restart the display nothing happens, just e black screen and red blinking LED. Anyone has an idea why? does the software not work with the touch version? Thanks for your help

**Comment by bitcoinservicesag** [comment]
nobody with experience in this?

**Comment by rhumba99** [comment]
Did you solve it? I'm having the same problem.

**Comment by bitcoinservicesag** [comment]
Hi unfortunately not, nobody got back about it. Let me know if you find a solution i tried quite a lot. Greets Michael

**Reply by Py-Jinx** [reply]
I tried to make it run on the amoled touch, but have only limited knowledge - I have to admit.
I got it run with upload directly from platformio. 
You could also try it with my fork below:
https://github.com/Py-Jinx/NerdMiner_v2_w_S3_Amoled_Touch

**Comment by Py-Jinx** [comment]
I'm having the same issue, would appreciate some help / advice.

**Comment by jsy-rg1570** [comment]
I don't have the hardware to adapt the firmware. Your screen is off because it should been switched on using GPIO38. The firmware should be adapted about touch versus buttons. The actual AMOLED firmware is for the non touch version.

**Reply by bitcoinservicesag** [reply]
I would gift you one device if you could make it run :)

**Reply by jsy-rg1570** [reply]
> I would gift you one device if you could make it run :)
> 
> 

Done, you can check my pull request 😀 
https://github.com/BitMaker-hub/NerdMiner_v2/pull/334

**Reply by bitcoinservicesag** [reply]
Amazing looks good on your photos. I tired it also but i get this error message. Did you have the same?

<img width="1185" alt="Screenshot 2024-04-28 at 10 37 23" src="https://github.com/BitMaker-hub/NerdMiner_v2/assets/114337198/ea5f405f-29a9-460a-8230-45aef90eca7f">
